CHANGELOG.md

```diff
@@ -154,6 +154,9 @@ All notable changes to this project will be documented in this file.
 [`explicit_counter_loop`]: https://github.com/Manishearth/rust-clippy/wiki#explicit_counter_loop
 [`explicit_iter_loop`]: https://github.com/Manishearth/rust-clippy/wiki#explicit_iter_loop
 [`extend_from_slice`]: https://github.com/Manishearth/rust-clippy/wiki#extend_from_slice
+[`filter_flat_map`]: https://github.com/Manishearth/rust-clippy/wiki#filter_flat_map
+[`filter_map`]: https://github.com/Manishearth/rust-clippy/wiki#filter_map
+[`filter_map_flat_map`]: https://github.com/Manishearth/rust-clippy/wiki#filter_map_flat_map
```

Comment:
I also think that this should be one lint instead of three. I cannot think oft a Situation where we'd want different levels for them which is literally the only valid reason to split Lint's.